data-raw/lingcod_recreational_comps.R

```diff
@@ -42,15 +43,17 @@ age_bin = seq(0, 20, 1)
 #General Recfin
 ###############
 #Zip "pulled_4_19_21" folder from Lingcod_2021>data-raw>recfin into data-raw folder in repository
-recfin_bio = read.csv(file.path(dir,"Lingcod_2021","data-raw","pulled_4_19_21","SD501--2001---2020_rec_bio_lingcod_pulled_4_19_21.csv"))
+# PLEASE get this file from google drive in the data-raw folder rather than saving it in a subdirectory
+# people should only have to download files at the top level of data-raw
+recfin_bio = read.csv(file.path(dir, "data-raw", "SD501--2001---2020_rec_bio_lingcod_pulled_4_19_21.csv"))
```

Comment:
I dont understand this comment since we need the data accessibile somewhere. Would you prefer accessing directly from the google drive?

Reply:
if a file is being used it should have a link to it in the google drive under data-raw\ so people don't have to download every subdirectory or search for things in sub directories. my thought was, if you didn't have any data, you could just select all the files in data-raw without having to download everything in the sub directories.

note that i don't necessarily think this is the best option, but it is at least consistent. i am happy to brainstorm ideas for future versions. i started some notes for our futures selves [here](https://docs.google.com/document/d/1mtwUsE_RQaLPBbQ174x_19tpsLkjhfMRxWEZR3bjiTg/edit?usp=sharing).

Reply:
What do you mean by a "link" to it? Comment within the script saying its from the google drive an active link within the google drive folder?

Reply:
Right click in google drive and select "Add shortcut to Drive" then navigate to data-raw